BlazarData/src/main/resources/schema.sql

```diff
@@ -1,197 +1,193 @@
 --liquibase formatted sql
```

Comment:
Use quotes only when they are needed and also use a quote type that is compliant with other SQL DBs besides mysql so that other DBs can be potentially used.